test/scripts/e2e_go_tests.sh

```diff
@@ -91,14 +91,7 @@ fi
 
 echo "PARALLEL_FLAG = ${PARALLEL_FLAG}"
 
-if [ "$USE_CIRCLECI_SPLIT" != "" ]; then
-    PARALLEL_FLAG="-p 1"
-    PACKAGES="$(go list ./... | circleci tests split --split-by=timings --timings-type=classname)"
-    PACKAGE_NAMES=$(echo $PACKAGES | tr -d '\n')
-    echo "Testing the following packages:"
-    echo $PACKAGE_NAMES
-    gotestsum --format pkgname --junitfile $TEST_RESULTS/results.xml --jsonfile $TEST_RESULTS/testresults.json -- ${RACE_OPTION} ${PARALLEL_FLAG} -timeout 1h -v ${SHORTTEST} ${PACKAGE_NAMES}
```

Comment:
Unfortunately, I don't think this change is quite right. `GOTESTCOMMAND` isn't being set and it defaults to `go test`, so after this change we'll revert back to `go test`.  If we switch to calling `make integration` instead of no longer be setting `./test/scripts/run_integration_tests.sh`, then `GOTESTCOMMAND` will be set but wont include `--junitfile $TEST_RESULTS/results.xml --jsonfile $TEST_RESULTS/testresults.json`.

Reply:
This is some dead code and so this code path was not being exercised because USE_CIRCLECI_SPLIT is not set anywhere. This commit should have no effect on the tests we're running in CircleCI.

I'm not sure how running make targets will work after attaching a workspace saved by a different job.